src/Mobs/Monster.h

```diff
@@ -42,7 +42,7 @@ class cMonster:
 
 	/** Creates the mob object.
 	If a_ConfigName is not empty, the configuration is loaded using GetMonsterConfig()
-	a_MobType is the type of the mob (also used in the protocol ( http://wiki.vg/Entities#Mobs 2012_12_22))
+	a_MobType is the type of the mob (also used in the protocol ( http://wiki.vg/Entities#Mobs 2012_12_22)) Page no longer exists 09.12.2024
```

Comment:
As per https://web.archive.org/web/20241130182632/http://wiki.vg/Entities this redirects to Entity metadata: https://minecraft.wiki/w/Minecraft_Wiki:Projects/wiki.vg_merge/Entity_metadata

Due to changes in the protocol in more recent versions, mobs/monsters are no longer spawned different from other entities, so the current version of that page does not have a "Mobs" section anymore.
I'd say: Either link to some old version of Entity_metdata page (edit history is still available!) or find suitable other page.